lms/static/js/edxnotes/plugins/search_override.js

```diff
@@ -0,0 +1,14 @@
+(function(define, undefined) {
+    'use strict';
+
+    define(['annotator_1.2.9'], function(Annotator) {
+    /**
```

Comment:
Nit: I'd use ``//`` for this comment, and indent it four spaces to match the code it is commenting.